api_tests/src/actor_test.ts

```diff
@@ -24,7 +24,7 @@ function nActorTest(
         const actors = await createActors(name, actorNames);
 
         try {
-            await timeout(60_000, testFn(actors));
+            await timeout(120_000, testFn(actors));
```

Comment:
Is it really necessary to be that high? So far the tests ran just fine on my machine and with these high values we have to wait quite long if something goes wrong.

I assume that there is 1 or 2 tests which take longer than expected and these are probably the LN sanity tests. What do you think of making this value configurable and defaulting to `60_000` if not defined?

Reply:
Happy to look at making it configurable. Tests do not run "just fine" on my machine with lower value.